homeassistant/components/qbittorrent/client.py

```diff
@@ -0,0 +1,19 @@
+"""Some common functions used in the qbittorrent components."""
+from qbittorrent.client import Client
+
+
+def get_main_data_client(client: Client):
+    """Get the main data from the Qtorrent client."""
+    return client.sync_main_data()
+
+
+def create_client(url, username, password):
+    """Create the Qtorrent client."""
+    client = Client(url)
+    client.login(username, password)
+    return client
```

Comment:
This is used in a single place. I think it would make the code more readable if this was just moved into the `async_setup_entry` method directly.